lib/diagnostic.rb

```diff
@@ -105,15 +125,21 @@ def eat_meat(food)
 end
 
 ##
-# your response here
+class Lion < Cat
+  include Carnivorous
+
+  def roar
+    puts 'ROAR!'
+  end
+end
 ##
 
 # #
 # What are some of the advantages of using composition (i.e. mixins)
 # over using direct inheritance?
 
 ##
-# your response as a comment here
+# You can bring in behavior and not data.
 ##
```

Comment:
One can also bring in behavior with inheritance! The real power of mixins is being able to inherit that behavior from multiple sources, where as with inheritance we are limited to to inheriting from a single source.